variables.tf

```
@@ -27,6 +27,12 @@ variable "iam_permissions" {
   default     = ["ses:SendRawEmail"]
 }
 
+variable "iam_allowed_resources" {
+  type        = list(string)
+  description = "Specifies resource ARNs that are enabled for iam_permissions above. Wildcards are acceptable."
```

```suggestion
  description = "Specifies resource ARNs that are enabled for var.iam_permissions. Wildcards are acceptable."
```

I think this is a better way to reference another variable, just objectively with the `var.` prefix, and not mentioning where it might be in the `variables.tf` file. But that's just a matter of style, so not a big deal.